src/utils/ethutil.js

```diff
@@ -7,6 +6,11 @@ export function setWeb3(_web3) {
 }
 
 export function getTruffleContract(jsonABI, defaults = {}) {
+  //Doing this here instead of `import` so that the project uses the project version of Web3Js
```

Comment:
@xaler5 

Thanks for doing this! Just a couple of nits:

1. Add space after `//`
2. s/Web3Js/web3.js/

Will proceed to test using the preview link.